blinkpy/sync_module.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, blink, network_name, network_id, camera_list):
         self.available = False
         self.type_key_map = {
             "mini": "owls",
-            "lotus": "doorbell",
+            "lotus": "doorbells",
```

Comment:
You would have thought so, but the `homescreen` object uses `doorbells` and `doorbells` is also used in `get_camera_info` rather than using this mapping.

Reply:
I didn't want to do it in this PR but it might be worth changing `get_camera_info` to use that `type_key_map` as well.